src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -28,8 +28,11 @@
             + ": Lists information based on a given type.\n"
             + "Parameters: l/LIST_TYPE (must be either people, infected, quarantined, locations, visits,"
             + " high-risk-locations or stats)\n"
+            + "For high-risk-locations only:\n"
+            + "Parameters: HIGH_RISK_LOCATIONS_NUMBER l/high-risk-locations\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_LIST + "people";
+            + PREFIX_LIST + "people\n"
+            + "list 3 l/high-risk-locations";
 
```

Comment:
This set of message, seems to imply that high risk locations will always need a number, but I believe that you also allow for no number to be given as well? Perhaps you could wrap HIGH_RISK_LOCATIONS_NUMBER with [  ]
 brackets, since that is used to indicate that a parameter is optional.

In addition, I would suggest separating the two examples using "Ëxample: ".
Currently it would look like:
Example: list l/people
list 3 l/high-risk-locations.
It might look better like this.
Example: list l/people
Example: list 3 l/high-risk-locations.

Reply:
Fixed